typescript/widget-store/src/shop.ts

```diff
@@ -36,25 +36,24 @@ export const PRODUCT_ID = 1;
 
 const config = {
   client: 'pg',
-  connection: process.env.DBOS_DATABASE_URL ? process.env.DBOS_DATABASE_URL :
-  {
-    host: process.env.PGHOST || 'localhost',
-    port: parseInt(process.env.PGPORT || '5432'),
-    database: process.env.PGDATABASE || 'widget_store_node',
-    user: process.env.PGUSER || 'postgres',
-    password: process.env.PGPASSWORD || 'dbos',
-  },
+  connection: process.env.DBOS_DATABASE_URL
```

Comment:
do we like this approach better than generating a connection string if DBOS_DATABASE_URL isn't set? I used the following in the toolbox app.

```ts
const databaseUrl = process.env.DBOS_DATABASE_URL || 
  `postgresql://${process.env.PGUSER || 'postgres'}:${process.env.PGPASSWORD || 'dbos'}@${process.env.PGHOST || 'localhost'}:${process.env.PGPORT || '5432'}/${process.env.PGDATABASE || 'dbos_node_toolbox'}`;
```

we should be consistent between the demo apps. The approach here does seem easy to read.

You should also add a comment regarding the use of DBOS_DATABASE_URL for DBOS Cloud compat